terraform/cloud-platform/templates/kops.yaml.tpl

```diff
@@ -313,10 +313,8 @@ spec:
   maxSize: 1
   minSize: 1
   cloudLabels:
-    application: justice-cloud-platform
-    business-unit: hq
-    component: master
```

Comment:
Did you mean to remove the `component`s? I think it's actually good that we had those…

Reply:
It's a duplicate tag, already defined by kops under `kops.k8s.io/instancegroup`